public/dorequest.php

```diff
@@ -162,25 +165,30 @@ function DoRequestError(string $error, ?int $status = 200, ?string $code = null)
         }
         break;
 
+    // TODO require authentication?
     case "codenotes2":
         $response['CodeNotes'] = getCodeNotesData($gameID);
         $response['GameID'] = $gameID;
         break;
+
     case "gameid":
         $md5 = request()->input('m') ?? '';
         $response['GameID'] = getGameIDFromMD5($md5);
         break;
 
+    // TODO require authentication?
     case "gameslist":
         $consoleID = (int) request()->input('c', 0);
         $response['Response'] = getGamesListDataNamesOnly($consoleID);
         break;
 
+    // TODO require authentication?
     case "officialgameslist":
         $consoleID = (int) request()->input('c', 0);
         $response['Response'] = getGamesListDataNamesOnly($consoleID, true);
         break;
 
+    // TODO deprecate?
```

Comment:
the functionality that used to rely on this was removed from the DLL in 0.76 (2019)